README.md

```diff
@@ -6,4 +6,4 @@ This module provides a Python script for connecting to Discord using Webhooks.
 
 ## How to Use This Script 
 
-// TBD
+To use the module, clone this repository to your local computer and install it by navigating to the downloaded directry and then use <code>pip install .</code> to install the module in your environment.
```

Comment:
Please also write more explanations how to use your API. For example, provide a code snippet with classes.

Reply:
I have rewritten **readme.md**, please check it one more time when you get a chance. If there is something else, you can tell me in this pull request.